11LessonEleven/in-class-project/css/style.css

```diff
@@ -0,0 +1,92 @@
+body {
+    color: white;
+    margin: 30px;
+}
+
+header {
+    background-color: #2BCDC0;
+    height: 280px;
+    width: 600px;
+}
+
+h1 {
+    padding-top: 120px;
+    padding-left: 150px;
+    font-weight: normal;
+}
+
+main {
+    background-color: black;
+    height: 280px;
+    width: 600px;
+    text-align:center;
+}
+
+section {
+    height: 280px;
+    width: 600px;
+    padding-left: 50px;
+
+}
+
+div {
+    float: left;
+    height: 200px;
+    width: 120px;
+    padding: 20px;
+}
+
+h2 {
+  border-top: 20px solid white;
+  font-size: 20px;
+  padding-top: 15px;
+}
+
+p {
+  font-size: 12px;
+  }
+
+.blue h2 {
+  border-top: 20px solid #33A8DD;
+}
+
+.red h2 {
+  border-top: 20px solid #F46731;
+}
+
+.teal h2 {
+  border-top: 20px solid #2BCDBF;
+}
+
+@media (max-width: 767px) {
+  header {
+    width: 100%;
+    background-color: #33A8DD;
+    text-align: left;
+    padding-top: 100px;
+    height: 200px;
+  }
+
+  main {
+    color: #F7F5F6;
+  }
+
+  h1     {
+    font-size: 2.5em;
```

Comment:
Better to use the same type of dimensions in all your CSS: `px` instead of `em`. Generally speaking it's best to just use `px` - it's the standard.